accounts/ledger_account.md

```diff
@@ -0,0 +1,23 @@
+## LedgerAccount
+
+```
+class LedgerAccount:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+    // Suggestions:
+    constructor(account_index: int = 0, address_index: int = 0);
+
+    address: Address;
+
+    // Local copy of the account nonce.
+    // Must be explicitly managed by client code.
+    nonce: uint64;
+
+    // serializes the transaction, signs the transaction, returns the signature;
+    sign_transaction(transaction: Transaction, account_index: int = 0, address_index: int = 0, sign_using_hash: boolean = True): bytes;
```

Comment:
Should follow `IAccount` with no alteration.

Developer is responsible to call `set_address` separately, on ledger app object, maybe?

Reply:
Or, maybe, provide an additional function `set_address`, which calls `self.ledger_app.set_address()`?

Reply:
added